zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/ProcessingDbState.java

```diff
@@ -75,15 +75,14 @@
 
 public class ProcessingDbState implements MutableProcessingState {
 
+  final TransientPendingSubscriptionState transientProcessMessageSubscriptionState;
```

Comment:
🔧 Let's make this private
🔧 Let's place this at the bottom of this state fields. All other state classes are more relevant to this class then this transient state.